pkg/tsdb/azuremonitor/metrics/azuremonitor-datasource.go

```diff
@@ -368,6 +383,10 @@ func getQueryUrl(query *types.AzureMonitorQuery, azurePortalUrl string) (string,
 		return "", err
 	}
 	escapedChart := url.QueryEscape(string(chartDef))
+	// Azure Portal will timeout if the chart definition includes a space character encoded as '+'.
+	// url.QueryEscape encodes spaces as '+'.
+	// Note: this will not encode '+' literals as those are already encoded as '%2B' by url.QueryEscape
+	escapedChart = strings.ReplaceAll(escapedChart, "+", "%20")
```

Comment:
by the way, for this part is it returning errors on current versions? was there an issue for this? (is this a separate fix that should be backported?)